### ISSUE TITLE
Critical Vulnerability CVE-2021-41816 on cgi

### ISSUE THREAD

**Issue by moharana-subhashree** [treasure-data/omnibus-td-agent#290]
Hi,
1) I am using td-agent 4.2.0 rpm in k8s environment. Default gem cgi version is 0.1.0  and Ruby version is 2.7.4.
CVE-2021-41816 is reported on it. Can you please help me understand the impact of this vulnerability on td-agent 4.2.0 version ?

2) Also checked cgi version in td-agent 4.3.0 rpm where cgi version found to be 0.1.0.1 , is this version has the fix for  CVE-2021-41816 , because I don't find cgi  0.1.0.1 version in https://github.com/ruby/cgi/tags.

**Comment by ashie** [comment]
>     2. Also checked cgi version in td-agent 4.3.0 rpm where cgi version found to be 0.1.0.1 , is this version has the fix for  [CVE-2021-41816](https://github.com/advisories/GHSA-5cqm-crxm-6qpv) , because I don't find cgi  0.1.0.1 version in https://github.com/ruby/cgi/tags.

Yes, td-agent 4.3.0 doesn't affect it because it uses Ruby 2.7.5:
https://www.ruby-lang.org/en/news/2021/11/24/ruby-2-7-5-released/

>     1. I am using td-agent 4.2.0 rpm in k8s environment. Default gem cgi version is 0.1.0  and Ruby version is 2.7.4.
>        [CVE-2021-41816](https://github.com/advisories/GHSA-5cqm-crxm-6qpv) is reported on it. Can you please help me understand the impact of this vulnerability on td-agent 4.2.0 version ?

I'm not sure yet but it's not affect because it seems that no actual code in td-agent 4.2.0 uses `CGI.escape_html` or  `CGI.escapeHTML`.

BTW please report issues for td-agent 4 to https://github.com/fluent/fluent-package-builder/ instead of here.
